tool/tctl/common/collection.go

```diff
@@ -575,6 +575,7 @@ func (c *uiConfigCollection) resources() (r []types.Resource) {
 func (c *uiConfigCollection) writeText(w io.Writer, verbose bool) error {
 	t := asciitable.MakeTable([]string{"Scrollback Lines"})
 	t.AddRow([]string{string(c.uiconfig.GetScrollbackLines())})
+	t.AddRow([]string{string(c.uiconfig.GetShowResources())})
 	_, err := t.AsBuffer().WriteTo(w)
 	return trace.Wrap(err)
```

Comment:
```suggestion
	t := asciitable.MakeTable([]string{"Scrollback Lines", "Show Resources"})
	t.AddRow([]string{string(c.uiconfig.GetScrollbackLines()), string(c.uiconfig.GetShowResources())})
	_, err := t.AsBuffer().WriteTo(w)
	return trace.Wrap(err)
```

Or some other wording
But the current output doesn't seem correct to me :thinking:

Reply:
Updated. 
```
Scrollback Lines Show Resources  
---------------- --------------- 
499              accessible_only 
```